src/containers/ProductContainer.tsx

```diff
@@ -0,0 +1,51 @@
+import { IProduct } from "../interfaces";
+import { Product } from "../pages";
+
+const data1: Array<IProduct> = [
```

Comment:
Avoid naming meaningless term for variable, whether it's fake data or anything unofficial.